charts/kaito/workspace/templates/clusterrole.yaml

```diff
@@ -39,7 +39,10 @@ rules:
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["validatingwebhookconfigurations"]
     verbs: ["get","list","watch"]
+  - apiGroups: [ "batch" ]
+    resources: [ "jobs" ]
+    verbs: [ "get", "list", "watch", "create", "delete", "update", "patch" ]
```

Comment:
Do we need create, delete, update, and patch roles?

Reply:
removed

Reply:
Upon further debugging we actually need these roles - just like the deployments and statefulsets workloads managed by the controller. So is job for fine tuning. If user for example deletes their job controller needs permission to recreate it.